packages/cashier-v2/src/components/CashierLayout/CashierLayout.tsx

```diff
@@ -0,0 +1,28 @@
+import React from 'react';
+import { useDevice } from '@deriv-com/ui';
+import type { TSideNotes } from '../../types';
+import styles from './CashierLayout.module.scss';
+
+type TProps = {
+    children: React.ReactNode;
+    sideNotes: TSideNotes;
+};
+
+const CashierLayout: React.FC<TProps> = ({ children, sideNotes }) => {
+    const { isMobile } = useDevice();
+    const isTopMobileSideNotesVisible = isMobile && sideNotes.position === 'top' && sideNotes.notes.length > 0;
+    const isBottomMobileSideNotesVisible = isMobile && sideNotes.position === 'bottom' && sideNotes.notes.length > 0;
+
+    return (
+        <div className={styles.container}>
```

Comment:
my brain is getting confused all the time by not seeing component-specific names here, but it actually looks very clean